packages/core/src/utilities/validation.test.js

```diff
@@ -261,6 +261,41 @@ describe("fallsWithinNumericalRange", () => {
     ).toBeUndefined();
   });
 
+  test("succeeds with null", () => {
+    expect(
+      // $FlowFixMe - Typing should prevent this, but we're testing the output
+      fallsWithinNumericalRange({
+        value: null,
+        min: 0,
+        max: 5,
+        message: "Fail"
+      })
+    ).toBeUndefined();
+  });
+  test("succeeds with string null", () => {
+    expect(
+      // $FlowFixMe - Typing should prevent this, but we're testing the output
+      fallsWithinNumericalRange({
+        value: null,
```

Comment:
Shouldn't this be a string? Isn't the test exactly the same as that above otherwise?

Reply:
my mistake, thanks, fixed